tests/php/sync/test_class.jetpack-sync-modules-stats.php

```diff
@@ -4,12 +4,12 @@
 class WP_Test_Jetpack_Sync_Module_Stats extends WP_Test_Jetpack_Sync_Base {
 
 	function test_sends_stats_data_on_heartbeat() {
-		$this->markTestIncomplete( "Doesn't scale for large multisites" );
 		$heartbeat = Jetpack_Heartbeat::init();
 		$heartbeat->cron_exec();
 		$this->sender->do_sync();
 
 		$action = $this->server_event_storage->get_most_recent_event( 'jetpack_sync_heartbeat_stats' );
 		$this->assertEquals( JETPACK__VERSION, $action->args[0]['version'] );
+		$this->assertTrue( is_int( $action->args[0]['users'] ) );
```

Comment:
Might be nice to add a couple of users and assert that the count is correct, and also test for multisite, since the implementation is clearly trying to be accurate for multisite installs.